frontend/src/models/dashboardsModel.tsx

```diff
@@ -16,6 +16,15 @@ export const dashboardsModel = kea<dashboardsModelType>({
         // this is moved out of dashboardLogic, so that you can click "undo" on a item move when already
         // on another dashboard - both dashboards can listen to and share this event, even if one is not yet mounted
         updateDashboardItem: (item: DashboardItemType) => ({ item }),
+        updateDashboardRefreshStatus: (
```

Comment:
Yup definitely confusing, but this is intentional since we don't want to mount dashboardLogic in places where we mount dashboardsModel (i.e., renameDashboardItemSuccess, updateDashboardItem, etc.). Unfortunately, we already have this code pattern for updateDashboardItem  I'll add a separate comment like the one we already have on 16 and 17 to explain why this action is supposed to live here.